docs/changelog.md

```diff
@@ -11,6 +11,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
 
+* **concept-engine**:
+  * Added support for `SelectAddress` prompt request.
```

Comment:
Can you also add a reference to the related issue? This just helps when people want to get more context.